src/js/components/Dashboard.js

```diff
@@ -205,10 +205,12 @@ class Dashboard extends Component {
       <div ref="content" className="dashboard">
         <Header direction="row" justify="between" large={true} pad={{horizontal: 'medium'}}>
           {title}
-          <Search ref="search" inline={true} className="flex"
-            placeHolder="Search"
-            defaultValue={searchText} onChange={this._onSearch}
-            suggestions={searchSuggestions} />
+          <Box direction="row" pad={!navActive ? {horizontal: 'medium'} : 'none'} className="flex">
```

Comment:
`pad={{between: 'small'}}` on header works much nicer. 

Should I decrease the left padding for Search input within `index.scss`?